tests/api/test_checkout.py

```diff
@@ -434,6 +434,93 @@ def test_checkout_available_shipping_methods(
     assert data["availableShippingMethods"] == [{"name": shipping_method.name}]
 
 
+def test_checkout_available_shipping_methods_with_net_price_displayed(
+    monkeypatch, api_client, checkout_with_item, address, shipping_zone, site_settings
+):
+    shipping_method = shipping_zone.shipping_methods.first()
+
+    apply_taxes_to_shipping_mock = mock.Mock()
+    apply_taxes_to_shipping_mock.return_value = TaxedMoney(
+        net=shipping_method.price, gross=Money(13, "USD")
+    )
+
+    manager_mock = mock.MagicMock(apply_taxes_to_shipping=apply_taxes_to_shipping_mock)
+    monkeypatch.setattr(
+        "saleor.graphql.checkout.types.get_extensions_manager",
+        lambda **kwargs: manager_mock,
+    )
```

Comment:
`monkeypatch.setattr(ExtensionsManager, "calculate_checkout_subtotal", Mock(return_value=subtotal))`
You can do this in one line